ci_support/run_docker_build.sh

```diff
@@ -50,58 +50,58 @@ conda info
     export CONDA_PY=27
     set +x
     conda build /recipe_root --quiet || exit 1
-    
+
     /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
-    
```

Comment:
I don't know if this whitepsace removal is going to end nicely... we may get a PR which "fixes" it.